src/comment.rs

```diff
@@ -656,9 +656,16 @@ impl<'a> CommentRewrite<'a> {
                     _ => {
                         let mut config = self.fmt.config.clone();
                         config.set().wrap_comments(false);
-                        match crate::format_code_block(&self.code_block_buffer, &config) {
-                            Some(ref s) => trim_custom_comment_prefix(&s.snippet),
-                            None => trim_custom_comment_prefix(&self.code_block_buffer),
+                        if config.format_doc_comments() {
```

Comment:
I don't know how to write this if block a better way, please give me suggestion.

Reply:
And I wonder how to select multiply lines in the commit view. please give me suggestion.